prac_05/state_names.py

```diff
@@ -0,0 +1,23 @@
+"""
+CP1404/CP5632 Practical
+State names in a dictionary
+File needs reformatting
+"""
+from sys import excepthook
```

Comment:
Did you have an older version of Python? these two inputs don't appear in my coding...